### PR TITLE
Gtk for red-0.6.4

### PR DESCRIPTION
**Pull request by rcqls**
Updated code to be used at least with red-0.6.4

### CODE REVIEW

File: build/includes.r [hunk 1]
```diff
@@ -157,6 +157,7 @@ write %build/bin/sources.r set-cache [
 			%win32.reds
 			%COM.reds
 			%image-gdiplus.reds
+			%image-gdk.reds
```
Comment (qtxie) [inline build/includes.r]: This file is included twice.

File: modules/view/backends/platform.red [hunk 2]
```diff
@@ -790,18 +794,18 @@ system/view/platform: context [
 					progress:	21
 				]
 			]
+			; GTK Backend (similar to macOS from now)
 			Linux	[
 				extend system/view/metrics/def-heights [
```
Comment (qtxie) [inline modules/view/backends/platform.red]: Is there a reason to change those def-heights to simliar to macOS? IIRC, those values are chosen specific for GTK3.

File: build/includes.r (hunk 1 above)
Comment (rcqls) [reply build/includes.r]: For the rules.red file I 'll only keep the Cancel-OK rule.

File: modules/view/backends/platform.red (hunk 2 above)
Comment (rcqls) [reply modules/view/backends/platform.red]: This is here as a reminder to fix the values later. You know that this branch is far from being stable.

Comment (qtxie) [reply modules/view/backends/platform.red]: But those values are chosen on purpose, not arbitrary. Does the new values gives better results?

Comment (rcqls) [reply modules/view/backends/platform.red]: Apparently, no significant effect. I then removed this setting

**Pull request merged**